src/content/contributing/style-guide/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Style Guide
+description: Style Guide for ethereum.org
+lang: en
+sidebar: true
+---
+
+# Ethereum.org Style Guide {#style-guide}
+
+Content on [Etheruem.org](/) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
```

Comment:
Another small convention we've been using throughout the site is not capitalizing the website name of ethereum.org (unless it starts a sentence).
```suggestion
Content on [etheruem.org](/) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
```